tools/dm/cluster-operations.md

```diff
@@ -40,7 +40,7 @@ This sections describes the considerations that you need to know when you restar
 
 **In the process of full data loading:**
 
-For the SQL files during full data import, DM uses the downstream database to record the checkpoint information. When DM-worker is restarted, it checks the checkpoint information and you can use the [`start-task` command](../tools/data-migration-practice.md#step-4-start-the-data-synchronization-task) to recover the data synchronization task automatically.
+For the SQL files during full data import, DM uses the downstream database to record the checkpoint information. When DM-worker is restarted, it checks the checkpoint information and you can use the [`start-task` command](/tools/dm/practice.md#step-4-start-the-data-synchronization-task) to recover the data synchronization task automatically.
```

Comment:
```suggestion
For the SQL files during full data import, DM uses the downstream database to record the checkpoint information. When DM-worker is restarted, it checks the checkpoint information and you can use the [`start-task` command](../../tools/dm/practice.md#step-4-start-the-data-synchronization-task) to recover the data synchronization task automatically.
```

Reply:
**Any relative path** that links to **another file** must start from the level of the root directory. Otherwise, your link will be a broken link at the PingCAP website.

Please update all related links involved in this PR. :)

Reply:
can we refine the script for PingCAP website?

Reply:
@GregoryIan I checked with @xuechunL. We do support the type of links in this PR. So you can just keep them. :)